src/Core/dataentry.cpp

```diff
@@ -23,7 +23,7 @@ std::streamsize ShadyCore::StreamFilter::xsgetn(char_type* buffer, std::streamsi
 	}
 	pos += buffered;
 
-	count = base->sgetn(buffer + buffered, min(count, (std::streamsize)(size - pos)));
+	count = base->sgetn(buffer + buffered, std::min(count, (std::streamsize)(size - pos)));
```

Comment:
isn't the code below working on linux here?
```
#ifndef _MSC_VER
	using std::min;
	using std::max;
#endif
```

Reply:
I think it is, probably a left over from my first attempt at making it work